src/Modals.tsx

```diff
@@ -96,7 +97,7 @@ export const Modals = () => {
 
       {shouldShowPWAModal && <PwaModal />}
 
-      {isNotificationPwaModalOpen && <NotificationPwaModal />}
+      {!isCI && isNotificationPwaModalOpen && <NotificationPwaModal />}
```

Comment:
Since we don't have fully feature support for the notifications by Playwright, this might cause some issues on the tests for now. Lets disable notification permission modal on the CI.

- See https://github.com/microsoft/playwright/issues/3301